Doc/library/re.rst

```diff
@@ -572,11 +572,8 @@ character ``'$'``.
    Word boundaries are determined by the current locale
    if the :py:const:`~re.LOCALE` flag is used.
 
-   .. note::
-
-      Note that ``\B`` does not match an empty string, which differs from
-      RE implementations in other programming languages such as Perl.
-      This behavior is kept for compatibility reasons.
+   .. versionchanged:: 3.14
```

Comment:
```suggestion
   .. versionchanged:: next
```

Reply:
Docs are built continuously so features documented online may not be yet released in practice (only on HEAD). When we move to 3.14.0, then all next will be replaced by 3.14. Until then, they will be replaced by 3.14.a3 (unreleased) for instance.